src/main/java/http_server/HttpServerSocket.java

```diff
@@ -0,0 +1,24 @@
+package http_server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class HttpServerSocket implements ServerSocketManager {
+
+    private final int port;
+
+    public HttpServerSocket(int port) {
+        this.port = port;
+    }
+
+    public ClientSocket accept() {
+        HttpSocket httpSocket = null;
+        try {
+            ServerSocket socketManager = new ServerSocket(port);
+            httpSocket = new HttpSocket(socketManager.accept());
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
I know we talked about this in the IPM but try to think about what you want to happen here, as with all the errors.  There should also be tests for what happens if an error is thrown.